.github/workflows/go-postsubmit.yml

```diff
@@ -0,0 +1,40 @@
+name: GoPostSubmit
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+  workflow_dispatch: {}
+
+env:
+  # Common versions
+  GO_VERSION: '1.16'
+  GO_REQUIRED_MIN_VERSION: ''
+  GOPATH: '/home/runner/work/cluster-proxy/cluster-proxy/go'
+defaults:
+  run:
+    working-directory: go/src/open-cluster-management.io/cluster-proxy
+
+jobs:
+  verify:
+    name: images
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          path: go/src/open-cluster-management.io/registration
+      - name: install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: install imagebuilder
```

Comment:
this may not be necessary if you have docker build command in `make images`.

Reply:
removed